### ISSUE TITLE
Bezier function draw split curves when specific points are inputted

### ISSUE THREAD

**Issue by Salleyi75** [p5py/p5#199]
**Describe the bug**
The bezier function creates a split curve when the following coords are inputted "bezier((85, 20), (10, 10), (90, 90), (15, 80))"

**To Reproduce**
```
from p5 import *

def draw():
    background(204)
    bezier((85, 20), (10, 10), (90, 90), (15, 80))

run()
```
run the above

**Expected behavior**
A smooth continuous curve is created, as shown below:
![image](https://user-images.githubusercontent.com/34999259/87831026-ed850980-c871-11ea-8c22-245c5457bdda.png)

**Screenshots**
The curve below is shown instead
![image](https://user-images.githubusercontent.com/34999259/87831169-35a42c00-c872-11ea-8ebb-cddc98c17a9c.png)

**System information:**
 - p5 release (version number or latest commit): https://github.com/p5py/p5/commit/baac18960d8fff6b879133e65e2b10dc1c5af467
 - Python version: 3.7.5
 - Operating system: Windows 10 (64 bit)

**Additional context**
None

**Comment by yogeshhk** [comment]
Getting error:

```
File "p5\sketch\Vispy2DRenderer\openglrenderer.py", line 354, in _transform_vertices
    return np.dot(np.dot(vertices, local_matrix.T), global_matrix.T)[:, :3]
```

This is happening because vertices are in 3 column format (x,y,1), whereas both transformation matrices are 4x4 (as they are generic 3D homogenous transformation matrices). So, there is mismatch.

One way to fix is to change:

`np.hstack([vertices, np.ones((len(vertices), 1))])
`
to

`np.hstack([vertices,  np.zeros((len(vertices), 1)), np.ones((len(vertices), 1))])
`
So that vertices become 4 columns (x,y,0,1)

This fixes the error as well as original issue of broken Bezier. Its looking fine with this fix.

**Comment by ziyaointl** [comment]
Thanks to @yogeshhk, the problem is fixed in #311. I'll close this issue for now, though another issue should probably be opened for the bezier curve not filling properly.

<img width="484" alt="Screen Shot 2021-05-31 at 11 40 54 PM" src="https://user-images.githubusercontent.com/18119047/120263434-80dbe900-c250-11eb-9951-76be89151df7.png">
